server/src/main/java/com/broadcom/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -117,7 +117,7 @@ CopybookModel resolveSync(
       @NonNull String documentUri,
       @NonNull CopybookProcessingMode copybookProcessingMode)
       throws ExecutionException, InterruptedException {
-    String cobolFileName = files.getNameFromURI(documentUri);
+    @NonNull String cobolFileName = files.getNameFromURI(documentUri);
```

Comment:
<img width="1142" alt="Screenshot 2020-11-12 at 15 25 29" src="https://user-images.githubusercontent.com/10369965/98958679-d1813400-2502-11eb-9ce7-886cc4f83411.png">

Reply:
Please, rewrite this too. The annotation will not work here as well.
As I see, this error is about null check between `computeIfAbsent` and `add` calls.